WIP: Initial 1.16.5 code push

maximuslotro opened the pull request:
Re-wrote all the client code.
Re-wrote most utils code.
Randomly re-wrote a bunch of server code.
updated mod info files to mod.toml

Can't Get gradle to build the client jar.
Need help re-writing the build.gradle script.

maximuslotro committed:
Major version changes

Update Gradle and build script
transition lang  file to json file for client
add mods.toml file for client
delete mcmod.info file for client
start updating main client code

maximuslotro committed:
Event updates

Updated some events (client)
Potentially added ability for client to tell if server is running forgeessentials

maximuslotro committed:
update gradle build properties

maximuslotro committed:
Get commands working on client

Port client command
start working on client networking

maximuslotro committed:
change main mod info file format

maximuslotro committed:
Bunch of forge redirecting

maximuslotro committed:
Bunch of Code Changing

maximuslotro committed:
Bump spongeapi version

maximuslotro committed:
More porting work

wow, its 3am, I should really get some sleep.

maximuslotro committed:
Code Changes

maximuslotro committed:
Update client command handler

maximuslotro committed:
Code Changes

maximuslotro committed:
update get server tps code

maximuslotro committed:
Update README.md

maximuslotro committed:
Update README.md

maximuslotro committed:
fixes for my file organizing

maximuslotro committed:
Many Fixes.

maximuslotro committed:
Code Change

maximuslotro committed:
code changes

maximuslotro committed:
MANY changes

maximuslotro committed:
Started Rework of network handler

maximuslotro committed:
More Networking and code changes

maximuslotro committed:
Dynamic Texture Loader

Due to DynamicTexture loading NativeImage instead of BufferedImage now, the Bufferedimage was converted into a bytearraystream and made into an image, the loaded by an InputStream into a NativeImage

maximuslotro committed:
Client Reach Handler Fix

Moved reach changing method to Client side code.

maximuslotro committed:
Many More Changes

Client Mod is almost free of errors!

maximuslotro committed:
Code Cleaning+ Porting

maximuslotro committed:
Changes

maximuslotro committed:
Fix Mixins and client perm overlay

maximuslotro committed:
dont know why this didnt push

maximuslotro committed:
Finish Permission overlay

maximuslotro committed:
Add Framework for config system

maximuslotro committed:
Add Config System

maximuslotro committed:
Code cleanup.

All errors removed from client code.

maximuslotro committed:
petty stuff

maximuslotro committed:
Fix gradle

maximuslotro committed:
add setup instructions

maximuslotro committed:
Updates license info

maximuslotro committed:
Move client files to separate folder

maximuslotro committed:
Move files again

maximuslotro committed:
Fix broken classes and mixins

maximuslotro committed:
Fixed mixins pointing to obsolete classes

maximuslotro commented:
Closed this pull because im making more changes that need to be added to my fork before this can be merged.

maximuslotro committed:
Fix some issues preventing mod from loading.

spacebuilder2020 commented:
BTW, you can just mark a PR as WIP and keep adding commits

maximuslotro committed:
Update config file data

maximuslotro committed:
Fix forge server network error

Fixed not being able to join a forge server that was not running the client mod. (why should the server be running it?)

maximuslotro committed:
Fix Incompatible FML modded server error

Set mod as a client mod, to fix server list "Incompatible FML modded server" error next all servers.

maximuslotro committed:
Change Command classes to appropriate folder

maximuslotro committed:
Clean-up work

maximuslotro committed:
Change to better message handler

maximuslotro committed:
Update README.md

maximuslotro committed:
Update README.md

maximuslotro committed:
Update README.md

maximuslotro committed:
Update README.md

maximuslotro committed:
Update README.md

maximuslotro committed:
Add total size

Total lines is broken from the host side

maximuslotro committed:
Nbt and registry updates

maximuslotro committed:
Js and Util Changes

maximuslotro committed:
Move Utils to appropriate folder

maximuslotro committed:
Lots of Changes

maximuslotro committed:
Many Changes

maximuslotro committed:
Some Stuff

maximuslotro committed:
Many Changes

maximuslotro committed:
Start work on new config system

maximuslotro committed:
Start work on new config system

maximuslotro committed:
Work on config system

maximuslotro committed:
Config work

maximuslotro committed:
Config+Misc changes

maximuslotro committed:
Use reflection to get current ModList containers

maximuslotro committed:
Changed code that was deprecated.

maximuslotro committed:
Misc Changes.

maximuslotro committed:
Update Auth handler

maximuslotro committed:
Config System Total Overhaul.

maximuslotro committed:
Fix method call being null

Fix trying to statically get a non static variable

maximuslotro committed:
Clean up

maximuslotro committed:
Fix EclipseFormatter xml error

Fixed Error: No grammar constraints (DTD or XML schema) detected for the document.

maximuslotro committed:
Changes

maximuslotro committed:
Made Mod Internally Server-side only

Will not show on the server list when a players pings the server as a mod they are missing.

maximuslotro committed:
Misc Changes.

maximuslotro committed:
Removed Baubles Dependency

Baubles's Last version was 1.5.2 for MC1.12.2. Removed un-updatable code.

maximuslotro committed:
Eclipse Format Code

maximuslotro committed:
Remove Un-needed info file

maximuslotro committed:
More changes

maximuslotro committed:
Update Worldedit files

maximuslotro committed:
Port worldedit compat

maximuslotro committed:
Irc Changes

maximuslotro committed:
Add baubles 1.16 back into compat

maximuslotro committed:
Re-Work a bit of the command system

maximuslotro committed:
Well with this new system, there are no retards

maximuslotro committed:
Semi-Undo Refractor

Accidentally Refactored A method name for the entire project not, not just in one class.  Mostly Undoing: e159fd395f27095d46fd24ef6a2b24781ee9470e

maximuslotro committed:
Misc command fixes

spacebuilder2020 committed:
Merge branch '1.16.5/initial' of https://github.com/maximuslotro/ForgeEssentials-1.16.5-PortAndRewrite into maximuslotro-1.16.5/initial

spacebuilder2020 committed:
Fix Gradle Errors

spacebuilder2020 committed:
Fix Syntax Errors caused by merge

spacebuilder2020 committed:
Couple More fixes

spacebuilder2020 committed:
Fix Case

maximuslotro committed:
Merge pull request #1 from ForgeEssentials/maximuslotro-1.16.5/initial

Maximuslotro 1.16.5/initial

maximuslotro committed:
Small Changes

The @Api annotation has been removed from forge.

spacebuilder2020 committed:
Merge branch '1.16.5/initial' into 1.16.5/initial

maximuslotro committed:
Misc commands

maximuslotro committed:
Removed old code that is already elsewhere

maximuslotro committed:
Merge branch '1.16.5/initial' of https://github.com/maximuslotro/ForgeEssentials-1.16.5-PortAndRewrite into 1.16.5/initial

maximuslotro committed:
Almost most of the config system

Merged EconomyConfig and Economy files into one config. Allowed reloading.

maximuslotro committed:
Metric porting

maximuslotro committed:
Work on module loading

Start working using the new system for getting classes with annotations. Rip ASMData.

maximuslotro committed:
Change command handling

Wow it is very easy to mess up on how to handle a command

maximuslotro committed:
Command fixes

maximuslotro committed:
Auth module fixes

Only commands need porting for auth module.

maximuslotro committed:
Changes

maximuslotro committed:
Remove FTB compat parts

I cant find a 1.16.5 FTBLib or FTBUtilities.

maximuslotro committed:
Ported Effects

maximuslotro committed:
Finish porting server Votifier

maximuslotro committed:
Finish data.v2 porting

maximuslotro committed:
Fixed missing import

javax.persistence was not importing automatically so I directly imported the jar file.

maximuslotro committed:
Update HelpFixer.java

maximuslotro committed:
Work on Sponge api

Had to add jar files for the imports because gradle.build not behaving.

maximuslotro committed:
Remove obsolete code and misc fixes

the command getUsage code is not used or needed anymore

maximuslotro committed:
Command and inventory stuff

maximuslotro committed:
fix fail client build

maximuslotro committed:
Update build.gradle

maximuslotro committed:
Update build.gradle

maximuslotro committed:
misc

see if key is fixed

maximuslotro committed:
dim work

maximuslotro committed:
Config updates

maximuslotro committed:
dim changes

maximuslotro committed:
Fix missing method names

maximuslotro committed:
Removed mcstats.org metrics

Mcstats.org has been offline for over 4 years now. Useless code.

maximuslotro committed:
Update method names

maximuslotro committed:
Fix missing com.google.inject.Inject

maximuslotro committed:
Misc changes

maximuslotro committed:
Commented out the multiworld module

maximuslotro committed:
Misc changes

maximuslotro committed:
Finished ModuleLauncher Port

maximuslotro committed:
remove obsolete imports

maximuslotro committed:
finish ASMData to AnnotationData port

maximuslotro committed:
Update ScriptArguments.java

maximuslotro committed:
Update TeleportHelper.java

spacebuilder2020 reviewed (approved):
Going to merge this As Is soon to keep the 1.16 port going along.

maximuslotro committed:
Dynamic Configs

Re-Wrote the config system to dynamically load configs based upon if a module is loaded or not.

maximuslotro committed:
Fix For eclipse Annotation Builder Breaking

maximuslotro committed:
Command System Rewrite Part:1

Re-Wrote the command system

maximuslotro committed:
Command System Rewrite Part:2

Re-wrote "chat" commands

maximuslotro committed:
Misc fixes + changes

maximuslotro committed:
Command System Rewrite Part:3

maximuslotro committed:
Update CONTRIBUTING.md

maximuslotro committed:
Merge branch '1.16.5/initial' of https://github.com/maximuslotro/ForgeEssentials-1.16.5-PortAndRewrite into 1.16.5/initial

maximuslotro committed:
Phase out ForgeEssentialsCommandBase

maximuslotro committed:
Command System Rewrite Part:4

maximuslotro committed:
Command System Rewrite Part:5

maximuslotro committed:
Command System Rewrite Part:6

Questioner Port

maximuslotro committed:
Command System Rewrite Part:7

Ticket Module port

maximuslotro committed:
Command System Rewrite Part:8

Port Mapper module and Perf module

maximuslotro committed:
Small changes

maximuslotro committed:
Command System Rewrite Part:9

maximuslotro committed:
Add custom Argument types

maximuslotro committed:
Command System Rewrite Part:10

maximuslotro committed:
Remove LatMod repo

maximuslotro committed:
Command System Rewrite Part:11

maximuslotro committed:
Change imports and clean docs

Moved the packed annotations jar (for IDEs), from the lib folder to the misc folder, so build tasks wont use it. Also updated the docs for annotations.

maximuslotro committed:
Update CONTRIBUTING.md

maximuslotro committed:
Command System Rewrite Part:11
